open-models/serving/cloud_run_ollama_gemma2_rag_qa.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>Do you want to set a variable here for the entire notebook here as in:</p><p>MODEL_NAME="gemma2:9b"</p><p> </p><p>This way, you can use the same variable later on when invoking Cloud Run.</p><p> </p><p>Also I'd perhaps remove the suggestion to try example <code>llama3</code> because the notebook's title refers only to Gemma 2. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Thanks, done!